db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package edu.artic.db.models
+
+/**
+ * @author Sameer Dhakal (Fuzz)
+ */
+
+fun ArticObject.getAudio(): ArticAudioFile? {
+    return if (this.audioCommentary.isNotEmpty()) {
+        this.audioCommentary.first().audioFile
```

Comment:
we could cleanup this method to:
```
fun ArticObject.getAudio(): ArticAudioFile? = this.audioCommentary.firstOrNull()?.audioFile
```

Reply:
also renaming it to an Extension property might be better:
```
val ArticObject.audioFile: ArticAudioFile? 
  get() = this.audioCommentary.firstOrNull()?.audioFile
```
And more semantic to Kotlin, not Java "getters".

Reply:
I implemented extension property.